### ISSUE TITLE
minor issue (probably only reporting)

### ISSUE THREAD

**Issue by albertocominelli** [OPM/opm-simulators#5787]
I am running some test cases for CO2STORE and I see some weird messages in the PRT file e.g.:
![Screenshot 2024-12-05 170759](https://github.com/user-attachments/assets/80f0b9c9-b4e2-4de3-9492-164b44615e5c)
Flo says some keyword are not supported. I suppose it is simply a matter of reporting a wrong piece of information.
Could you confirm this?ù
Regards,
A:

**Comment by albertocominelli** [comment]
BTW, I am using 2024.10

**Comment by totto82** [comment]
The CO2STORE implementation in OPM Flow is based on the blackoil simulator. The unsupported keywords in the list are compositional keyword, they are not needed and will be ignored. To setup a basic CO2STORE run you will need to use

CO2STORE
GAS
WATER
DISGASW (optional, if you would like to model dissolution of CO2 in brine)
VAPWAT (optional, if you would like to model vaporization of water)
THERMAL (optional)
DIFFUSE (optional)
in the RUNSPEC section. 

You will also need to set the salinity using the SALINITY keyword in the PROPS section. (i.e. you don't specify the salt components using COMPS)

**Comment by bska** [comment]
> The unsupported keywords in the list are compositional keyword, they are not needed and will be ignored.

Right, but this appears to be [`CO2STORE_GASWAT`](https://github.com/OPM/opm-tests/blob/6180545e5c0914cef09776985abfa39e0cee9690/co2store/CO2STORE_GASWAT.DATA) from OPM-Tests and that model uses those `COMPS`, `GASWAT` &c compositional keywords.  It's not really ideal if our own examples use misleading setup.
